ui/raidboss/data/03-hw/dungeon/sohm_al.txt

```diff
@@ -0,0 +1,141 @@
+119.6 "Flower Devour" sync /:Raskovnik:1392:/
+125.4 "Spit" sync /:Raskovnik:1393:/
+126.5 "--sync--" sync /:Raskovnik:1394:/
+136.0 "Floral Trap" sync /:Raskovnik:1391:/ jump 114.6
+141.0 "Flower Devour" # sync /:Raskovnik:1392:/
+146.8 "Spit" # sync /:Raskovnik:1393:/
+147.9 "--sync--" # sync /:Raskovnik:1394:/
+157.1 "Floral Trap" # sync /:Raskovnik:1391:/
+162.1 "Flower Devour" # sync /:Raskovnik:1392:/
+167.9 "Spit" # sync /:Raskovnik:1393:/
+# Loop
+200.0 "Sweet Scent" sync /:Raskovnik:1395:/ jump 103.0
+211.8 "Floral Trap" # sync /:Raskovnik:1391:/
+216.8 "Flower Devour" # sync /:Raskovnik:1392:/
+222.6 "Spit" # sync /:Raskovnik:1393:/
+223.7 "--sync--" # sync /:Raskovnik:1394:/
+
+# Group 2
+300.0 "--sync--" sync /:ED6:Raskovnik/ window 300,0
+303.0 "Leafstorm" sync /:Raskovnik:ED6:/
+308.2 "Bloody Caress" sync /:Raskovnik:ED1:/
+314.3 "--sync--" sync /:Raskovnik:ED2:/
+319.9 "Acid Rain" sync /:Raskovnik:ED7:/
+328.5 "Phytobeam" sync /:Raskovnik:ED5:/
+333.7 "Bloody Caress" sync /:Raskovnik:ED1:/
+# Loop
+342.8 "Leafstorm" sync /:Raskovnik:ED6:/ jump 303.0
+348.0 "Bloody Caress" # sync /:Raskovnik:ED1:/
+354.1 "--sync--" # sync /:Raskovnik:ED2:/
+359.7 "Acid Rain" # sync /:Raskovnik:ED7:/
+368.2 "Phytobeam" # sync /:Raskovnik:ED5:/
+373.4 "Bloody Caress" # sync /:Raskovnik:ED1:/
+
+# jump back to group 1
+500.0 "--sync--" sync /:1395:Raskovnik/ window 500,0 jump 103.0
+
+
+### Myath
+# -ic "Chyme Of The Mountain"
+
+1000.0 "Start" sync /00:0839:The Wound will be sealed off/ window 1000,1
+
+1006.1 "Third Leg Forward" sync /:Myath:1382:/
+1011.2 "Overbite" sync /:Myath:EDB:/
+1019.3 "Razor Scales" sync /:Myath:EDC:/
+1024.5 "Third Leg Forward" sync /:Myath:1382:/
+1031.6 "Third Leg Forward" sync /:Myath:1382:/
+1036.7 "Overbite" sync /:Myath:EDB:/
+# Loop
+1044.8 "Razor Scales" sync /:Myath:EDC:/ jump 1019.3
+1050.0 "Third Leg Forward" # sync /:Myath:1382:/
+1057.1 "Third Leg Forward" # sync /:Myath:1382:/
+1062.2 "Overbite" # sync /:Myath:EDB:/
+1070.3 "Razor Scales" # sync /:Myath:EDC:/
+1075.5 "Third Leg Forward" # sync /:Myath:1382:/
+
+# Myath would use Primordial Roar twice:
+# 1. HP < 90%?
+# 2. HP < 50%? also summon Chyme Of The Mountain
+1200.0 "Primordial Roar" sync /:Myath:EE2:/ window 200,0
+1207.1 "Third Leg Forward" sync /:Myath:1382:/
+1214.2 "Ensnare" sync /:Myath:EDD:/
+1219.2 "Mad Dash" sync /:Myath:EE[01]:/
+1221.5 "Ensnare" sync /:Myath:EDD:/
+1226.5 "Mad Dash" sync /:Myath:EE[01]:/
+1229.0 "Ensnare" sync /:Myath:EDD:/
+1236.1 "Mad Dash" sync /:Myath:EE[01]:/
+1238.4 "Ensnare" sync /:Myath:EDD:/
+1245.4 "Mad Dash" sync /:Myath:EE[01]:/
+
+# Jump back to loop
+1249.8 "Third Leg Forward" sync /:Myath:1382:/ window 30,5 jump 1006.1
+1254.9 "Overbite" # sync /:Myath:EDB:/
+1263.0 "Razor Scales" # sync /:Myath:EDC:/
+1268.2 "Third Leg Forward" # sync /:Myath:1382:/
+1275.3 "Third Leg Forward" # sync /:Myath:1382:/
+1280.4 "Overbite" # sync /:Myath:EDB:/
+
+
+### Tioman
+# -ic "Comet"
```

Comment:
Probably also `-p EE3:2006` and could you also run it with `-it Tioman` so you get the targetable/untargetable lines when Tioman disappears for Heavensfall? Were there any things you needed to `-ii` and ignore? (Similar questions about `-p` and `-ii` on the other timelines, although I think this is the only one with a disappearing boss.)

Reply:
Okay, I'll add them soon!

Reply:
`-it` didn't work because Tioman was just invincible, not disappear or untargetable...